### PR TITLE
feat: 30 run all fettle functions at once

### PR DESCRIPTION
**Pull request by mikeylive**

### CODE REVIEW

**Review by tylerkron — CHANGES_REQUESTED** [review]
A couple of comments

1. I get a handful of lint issues building this
image

2. Can you update the readme of how to run all the fettle functions?

**Comment by mikeylive** [comment]
Lint issues, did we investigate this? https://kotlinlang.org/docs/code-style-migration-guide.html  Not sure this is the right approach but I'll fix.

**Comment by JacobBrauchler** [comment]
> Lint issues, did we investigate this? https://kotlinlang.org/docs/code-style-migration-guide.html Not sure this is the right approach but I'll fix.

we are using ktlint if you look at the readme it will tell you the command to run to give us a precommit hook. then ktlint format runs every time. But no I didn't look at that.

**Comment by mikeylive** [comment]
Fixed...thanks for both being shitty reviewers so I can't mark things as out of date or resolved 🖕

**Comment by JacobBrauchler** [comment]
> Fixed...thanks for both being shitty reviewers so I can't mark things as out of date or resolved 🖕

I just replied I will dismiss as soon as I say its good 🖕

File: src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            CommandResult.Passed("PASS")
+        } else {
+            CommandResult.Failed("FAIL")
+        }
+    }
+
+    override fun dependabot(org: String, repo: String, branch: String): CommandResult {
+        return try {
+            api.getRepository("$org/$repo").getFileContent("/.github/dependabot.yml")
+            CommandResult.Passed("PASS")
```
Comment (JacobBrauchler) [inline src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt:22]: some say PASS and FAIL and some say PASSED and FAILED make it consistent and make a constant for fucks sake

Comment (mikeylive) [reply src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt:22]: It doesn't matter...yet. The only thing that matters to score is whether its CommandResult.Passed or CommandResult.Failed....the string within it is for later use when surely Shirly someone will want to know why it passed or failed.

Comment (mikeylive) [reply src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt]: Fixed.

File: src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt [hunk 2]
```diff
@@ -19,7 +19,7 @@ class GitHubFettleHandler(override val context: FettleContext) : FettleHandler {
     override fun dependabot(org: String, repo: String, branch: String): CommandResult {
         return try {
             api.getRepository("$org/$repo").getFileContent("/.github/dependabot.yml")
-            CommandResult.Passed("PASS")
+            CommandResult.Passed("")
```
Comment (JacobBrauchler) [inline src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt:22]: ???? 🖕

**Review by JacobBrauchler — CHANGES_REQUESTED** [review]

File: src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt (hunk 2 above)
Comment (JacobBrauchler) [reply src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt]: lets just make two constants and use them throughout the class 🖕

Comment (mikeylive) [reply src/main/kotlin/dev/fidil/fettle/handler/GitHubFettleHandler.kt]: replace didn't find the blanks....your fault 🖕

**Review by JacobBrauchler — APPROVED** [review]

**Pull request merged**